shibboleth/backends.py

```diff
@@ -50,7 +51,12 @@ def authenticate(self, request, remote_user, shib_meta):
                 """
                 user.set_unusable_password()
                 user.save()
-                user = self.configure_user(user)
+                args = (request, user)
+                try:
+                    inspect.getcallargs(self.configure_user, request, user)
```

Comment:
Thanks @m3brown. Looks like getcallargs is deprecated: https://docs.python.org/3/library/inspect.html#inspect.getcallargs. Could we switch to the Signature calls they suggest instead?

Reply:
Good find! Interesting, because I literally copied that code from Django’s source. I’ll update this PR shortly, and I probably should submit a PR to Django as well!